src/config/constants.ts

```diff
@@ -22,6 +22,7 @@ export const PULL_SECRET_INFO_LINK = CLUSTER_MANAGER_SITE_LINK;
 export const DEFAULT_OPENSHIFT_VERSION: OpenshiftVersionOptionType = {
```

Comment:
How about removing this and not provide any default for `getBugzillaLink`? It is outdated and not really used.

Reply:
Good point. The `DEFAULT_OPENSHIFT_VERSION` seized to server its original purpose.